layersvt/screenshot.cpp

```diff
@@ -1029,6 +1140,7 @@ VKAPI_ATTR VkResult VKAPI_CALL QueuePresentKHR(VkQueue queue, const VkPresentInf
             fileName = base + ".ppm";
 #else
             fileName = to_string(frameNumber) + ".ppm";
+            printf("Screen Capture file is: %s \n", fileName.c_str());
```

Comment:
This could result in lots of output. But I realize the value of printing out the name of the capture file. Not sure what best solution is.